ts/features/profileSettings/store/reducers/index.ts

```diff
@@ -38,7 +49,7 @@ const persistConfig: PersistConfig = {
   key: "profileSettings",
   storage: AsyncStorage,
   version: CURRENT_REDUX_PROFILE_SETTINGS_STORE_VERSION,
-  whitelist: ["showProfileBanner"]
+  whitelist: ["showProfileBanner", "hasUserAcknowledgedSettingsBanner"]
```

Comment:
What about adding a migration for the addition of this persisted field?
Something like it was done for `isItWalletTestEnabled`

```ts
// Adds it wallet section FF
  "27": (state: PersistedState) =>
    merge(state, {
      persistedPreferences: {
        isItWalletTestEnabled: false
      }
    }),
```
What do you think?

Reply:
yes, you're right! Thanks!